### PR TITLE
Second Pull Request

### PR DESCRIPTION
**Pull request by simonle0525**

### CODE REVIEW

File: car1/car1.ino [hunk 1]
```diff
@@ -6,92 +6,134 @@
...
+  //digitalWrite(ledPin, LOW);
 }
  
 void loop() {
   if(Serial.available() > 0){
     char letter = Serial.read();
     Serial.println(letter);
-  
-    if(letter == 'f'){
+
```
Comment (hungple) [inline car1/car1.ino:34]: Why it has to be upper case F?

Comment (simonle0525) [reply car1/car1.ino:34]: Because cell phone sends 'F' as forward.

File: car1/car1.ino [hunk 2]
```diff
@@ -6,92 +6,134 @@
...
+    else if(letter == 'w'){
+      digitalWrite(13, LOW);   
+    }
+
+    else if(letter == 'U'){
+      digitalWrite(12, HIGH);   
+    }
+
+    else if(letter == 'u'){
```
Comment (philliple0610) [inline car1/car1.ino:104]: Why is upper case HIGH and lower case LOW? First letter that is good, friendship, unity, caring, kind, youthful, outstanding, upper case.

Comment (simonle0525) [reply car1/car1.ino:104]: Because it can be.

**Comment by simonle0525** [comment]
@hungple Can you please approve with the PR?

**Review by hungple — APPROVED** [review]

File: car1/car1.ino (hunk 2 above)
Comment (philliple0610) [reply car1/car1.ino:104]: Friendship is 'F'

**Pull request merged**